packages/react/src/primitives/Divider/Divider.tsx

```diff
@@ -19,7 +19,6 @@ const DividerPrimitive: Primitive<DividerProps, 'hr'> = (
     ComponentClassName.Divider,
     classNameModifier(ComponentClassName.Divider, orientation),
     classNameModifier(ComponentClassName.Divider, size),
-    label ? ComponentClassName.DividerLabel : null,
```

Comment:
what does this have to do with bem?

Reply:
Nothing, just unnecessary. Was able to remove some CSS

Reply:
Are we sure it's not necessary? Think it's used in the federated signin component.

Reply:
I updated the CSS to remove that classname: https://github.com/aws-amplify/amplify-ui/pull/4474/files#diff-30ce68f480e6f1f48bb3548f6d48300e8bcb87b2c39e83774f81564c0d67ced0R16